stdlib/sql.go

```diff
@@ -795,6 +795,16 @@ func (r *Rows) Next(dest []driver.Value) error {
 					}
 					return d.Value()
 				}
+			case pgtype.XMLOID:
+				var d []byte
+				scanPlan := m.PlanScan(dataTypeOID, format, &d)
+				r.valueFuncs[i] = func(src []byte) (driver.Value, error) {
+					err := scanPlan.Scan(src, &d)
+					if err != nil {
+						return nil, err
+					}
+					return d, nil
+				}
```

Comment:
Not totally sure if this is necessary / correct -- we don't use the standard library but observed that JSON was registered this way, so I followed suit. Open to advice or correction.

Reply:
To be honest my memory is a little fuzzy on that. But I think what is happening is that ensures the type of value passed to a `sql.Scanner` is a `[]byte` instead of a `string`. Not sure why that was preferred. But it should be safe to do the same thing for xml.

Reply:
Ahh, understood